glossary.md

```diff
@@ -0,0 +1,10 @@
+You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
+
+* LGTM: "Looks good to me", commonly used to approve a code review.
+* PTAL: please take a look.
+* RSLGTM: "Rubber stamp looks good to me", used when the reviewer is merely granting OWNERS approval without doing a proper code review.
```

Comment:
```suggestion
* RSLGTM: "Rubber stamp looks good to me", used when the reviewer is merely granting OWNERS approval without doing a full code review.
```

Reply:
I like James's suggestions and would extend it a bit further:

```suggestion
* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing a full code review.
```